### DISCUSSION TITLE
Run exe under WIN10

### DISCUSSION THREAD

**Discussion by yzhang6708** [giorgk/ichnos#11] [Q&A] [answered]
I downloaded the most recent files. Then I run ichnos.exe -v in the folder of Dev. The following system error popped up:
_The code execution cannot proceed because msmpi.dll was not found. Reinstalling the program may fix the problem._

Do you know where I can get this dll file?

Thanks!

**Comment by giorgk** [comment, marked as answer]
This issue seems to be related to mpi. Do you have Microsoft MPI installed?

According to [this thread](https://stackoverflow.com/questions/35664628/msmpi-dll-error-message-in-visual-studio-c) you need to install or reinstall the Microsoft MPI. The same thread also provides a link to download the mpi 
https://learn.microsoft.com/en-us/message-passing-interface/microsoft-mpi?redirectedfrom=MSDN

**Reply by yzhang6708** [reply]
Thanks. It works!
